app/uk/gov/hmrc/apisubscriptionfields/repository/SubscriptionFieldsRepository.scala

```diff
@@ -75,8 +80,34 @@ class SubscriptionFieldsMongoRepository @Inject()(mongoDbProvider: MongoDbProvid
     )
   )
 
-  override def save(subscription: SubscriptionFields): Future[(SubscriptionFields, Boolean)] = {
-    save(subscription, selectorForSubscriptionFields(subscription))
+  override def saveAtomic(subscription: SubscriptionFields): Future[(SubscriptionFields, IsInsert)] = {
+
+    val selector = selectorForSubscriptionFields(subscription)
+    val entity = subscription
```

Comment:
it is not needed to create this `entity` variable
you can directly use `subscription`

Reply:
fixed